code/src/java/pcgen/system/Main.java

```diff
@@ -297,10 +297,26 @@ public static void loadProperties(boolean useGui)
 
 		//Existing PropertyContexts are registered here
 		PropertyContextFactory defaultFactory = PropertyContextFactory.getDefaultFactory();
-		defaultFactory.registerPropertyContext(PCGenSettings.getInstance());
+		PropertyContext settingscontext = PCGenSettings.getInstance();
+		defaultFactory.registerPropertyContext(settingscontext);
 		defaultFactory.registerPropertyContext(UIPropertyContext.getInstance());
 		defaultFactory.registerPropertyContext(LegacySettings.getInstance());
 		defaultFactory.loadPropertyContexts();
+		//Make savepath directory if it doesn't exist
+		String savepath = settingscontext.getProperty(PCGenSettings.PCG_SAVE_PATH);
+		File savepath_dir = new File(savepath);
+		if (!savepath_dir.exists() && !savepath_dir.isDirectory())
+		{
+			try
+			{
+				Logging.log(Level.INFO, "Making directory " + savepath_dir);
+				savepath_dir.mkdir();
+			}
+			catch (Exception e)
+			{
+				e.printStackTrace();
```

Comment:
Prefer logging rather than printing a stack trace.  This could also use try with resources I suspect.